src/main/java/mate/academy/spring/service/UserService.java

```diff
@@ -0,0 +1,12 @@
+package mate.academy.spring.service;
+
+import java.util.List;
+import mate.academy.spring.model.User;
+import org.springframework.stereotype.Service;
+
+@Service
+public interface UserService {
```

Comment:
is it the correct place for this annotation?

Reply:
No